mta/entities/analysis_results.py

```diff
@@ -26,15 +27,16 @@ class AnalysisResultsView(BaseLoggedInPage):
     analysis_results = AnalysisResults()
     confirm_delete = Button("Yes")
     cancel_delete = Button("No")
-    sort_analysis = Text(locator=".//th[contains(normalize-space(.), 'Analysis')]//i[1]")
+    sort_analysis = Text(locator=".//th[contains(normalize-space(.), 'Analysis')]//button")
 
     @property
     def is_displayed(self):
         return self.run_analysis_button.is_displayed and self.title.is_displayed
 
     def clear_search(self):
         """Clear search"""
-        self.search.fill("")
+        self.search.fill(" ")
+        self.browser.refresh()
```

Comment:
This is not required .
Browser.refresh makes the test to fail many time in different OS.

In widgetastic/__init__.py 

Make this change  
self.browser.send_keys(f"{Keys.CONTROL}+a,{Keys.DELETE}", self)

     def fill(self, value):
        current_value = self.value
        if value == current_value:
            return False
        # Clear and type everything
        self.browser.click(self)
        self.browser.send_keys(f"{Keys.CONTROL}+a,{Keys.DELETE}", self)
        self.browser.send_keys(value, self)
        return True